### ISSUE TITLE
the result seems strange

### ISSUE THREAD

**Issue by herbertchen1** [kentonl/e2e-coref#13]
[217400] loss=0.10, steps/s=5933.69

i run test_singleton at the 217400 epoch, only get the result as follows:

Identification of Mentions: Recall: (9392 / 19155) 49.03%       Precision: (9392 / 10875) 86.36%        F1: 62.55%
--------------------------------------------------------------------------
Coreference: Recall: (6130 / 14610) 41.95%      Precision: (6130 / 8390) 73.06% F1: 53.3%
--------------------------------------------------------------------------

Official result for bcub
version: 8.01 /home/chenbo/e2e-coref-master/conll-2012/scorer/v8.01/lib/CorScorer.pm
Repeated mention in the key: 656, 656 7290

====== TOTALS =======
Identification of Mentions: Recall: (9392 / 19155) 49.03%       Precision: (9392 / 10875) 86.36%        F1: 62.55%
--------------------------------------------------------------------------
Coreference: Recall: (5691.75228764641 / 19156) 29.71%  Precision: (6630.35763531834 / 10875) 60.96%    F1: 39.95%
--------------------------------------------------------------------------

Official result for ceafe
version: 8.01 /home/chenbo/e2e-coref-master/conll-2012/scorer/v8.01/lib/CorScorer.pm
Repeated mention in the key: 656, 656 7290

====== TOTALS =======
Identification of Mentions: Recall: (9392 / 19155) 49.03%       Precision: (9392 / 10875) 86.36%        F1: 62.55%
--------------------------------------------------------------------------
Coreference: Recall: (1271.20096754289 / 4546) 27.96%   Precision: (1271.20096754289 / 2485) 51.15%     F1: 36.15%

it's the right result? or i've do something wrong...

**Comment by herbertchen1** [comment]
in this directory '/conll-2012/v4/data/train/data/english/annotations' , i just find the bc directory , so maybe i got the wrong training data.....

**Comment by kentonl** [comment]
Yes, you'll need the data from the full OntoNotes corpus to reproduce the results in the paper.
